components/storage/blockdevices/COMPONENT_SPIF/mbed_lib.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Standard pin names like SPI_* should be defined in target. I would understand you defining something new like SPIF_MOSI or SD_MOSI etc, but not redefining SPI.

Reply:
Again as far as know those values are interpreted to MBED_CONF_SPIF_DERIVER_SPI_*